src/main/java/uk/gov/justice/digital/hmpps/csr/api/controllers/DetailController.kt

```diff
@@ -89,8 +92,11 @@ class DetailController(private val detailService: DetailService) {
   fun getModified(
     @PathVariable
     region: Int
-  ): List<DetailDto> =
-    detailService.getModified(region)
+  ): List<DetailDto> {
+    RegionContext.setRegion(region.toString())
```

Comment:
minor - would be better for the region to take an int rather than having to do a `toString` all the time